### PR TITLE
Make Istio Envoy log message generic

### PR DESCRIPTION
**Pull request by mbenmclean**
Provide a summary of the change and which issue (i.e. ticket) is fixed.
If there are any dependencies, for example on a PR in another repository, list those.

### CODE REVIEW

File: platform-operator/controllers/verrazzano/component/istio/istio_component.go [hunk 1]
```diff
@@ -397,7 +397,7 @@ func (i istioComponent) GetDependencies() []string {
 
 func (i istioComponent) PreUpgrade(context spi.ComponentContext) error {
 	if vzconfig.IsApplicationOperatorEnabled(context.ActualCR()) {
-		context.Log().Infof("Stopping WebLogic domains that are have Envoy 1.7.3 sidecar")
+		context.Log().Infof("Stopping WebLogic domains that are have old Envoy sidecar")
```
Comment (davisgomes) [inline platform-operator/controllers/verrazzano/component/istio/istio_component.go:400]: "that are have old" sounds a bit awkward. Maybe "that have old Envoy sidecars" would work better.

Comment (bobdonat) [inline platform-operator/controllers/verrazzano/component/istio/istio_component.go:400]: bad english in message

Comment (mbenmclean) [reply platform-operator/controllers/verrazzano/component/istio/istio_component.go]: fixed

Comment (mbenmclean) [reply platform-operator/controllers/verrazzano/component/istio/istio_component.go]: fixed

**Review by davisgomes — APPROVED** [review]

**Review by bobdonat — APPROVED** [review]

**Pull request merged**